pkg/kube/grants/init.go

```diff
@@ -0,0 +1,14 @@
+package grants
+
+import (
+	"github.com/skupperproject/skupper/pkg/kube"
+)
+
+func Initialise(controller *kube.Controller, currentNamespace string, watchNamespace string, config *GrantConfig, generator GrantResponse) func() {
```

Comment:
Have never ran into "initialise" before! :gb:

Reply:
I can change to Init? Or would Configure be clearer?

Reply:
Oh, don't mind me! Just caught my eye and ended up learning something new: en-us prefers ize over ise, but ise is the Oxford dictionary spelling.